src/standalone/Vegetation/canopy_boundary_fluxes.jl

```diff
@@ -173,7 +173,8 @@ function canopy_boundary_fluxes!(
     shf .= canopy_tf.shf
     lhf .= canopy_tf.lhf
     r_ae .= canopy_tf.r_ae
-
+    p.canopy.energy.∂LHF∂qc .= canopy_tf.∂LHF∂qc
```

Comment:
ideally we would set p.canopy.energy.var .= canopy_tf, but this will be a different PR. it would save an allocation